utils/users.js

```diff
@@ -47,6 +47,25 @@ const getUsername = async (userId) => {
   }
 };
 
+/**
```

Comment:
I have created a function in the user's util to fetch the first and last names, as there is no existing dedicated function to do so.
**Requirement of first and last name:** To display user's name in extension request logs
![image](https://github.com/Real-Dev-Squad/website-backend/assets/56365512/2b8d06cc-bc10-45e5-bf04-07bd22315baa)